addons/packages/app-toolkit/0.1.0/README.md

```diff
@@ -179,3 +187,21 @@ This example illustrates creating a simple Spring Boot web app on the locally de
     ```shell
     curl http://tanzu-simple-web-app.default.127-0-0-1.sslip.io/
     ```
+  
+## Troubleshooting
+
+Error: invalid configuration: no configuration has been provided
+
+* You must have a Tanzu unmanaged-cluster targetted to install App Toolkit
+
+Error: no matches for kind "PackageInstall" in version "packaging.carvel.dev/v1alpha1"
+
+* App Toolkit currently only supports Tanzu unmanaged-clusters. Double check that your current `kube config` is not pointing at an unsupported cluster type.
```

Comment:
@ryanmattcollins while that is true, we don't have anything in our product that conflicts with the underlying cluster provider (for example TCE managed clusters. We should remove this troubleshooting tip, as it may not really be useful for users to troubleshoot it.

Reply:
I created a new issue to capture this comment, #3924